autobump-core/src/main/java/com/github/autobump/core/model/usecases/CommentCreatedUseCase.java

```diff
@@ -0,0 +1,47 @@
+package com.github.autobump.core.model.usecases;
+
+import com.github.autobump.core.model.Setting;
+import com.github.autobump.core.model.SettingsRepository;
+import com.github.autobump.core.model.events.CommentCreatedEvent;
+import lombok.Builder;
+
+@Builder
+public class CommentCreatedUseCase {
+    SettingsRepository settingsRepository;
+
+    public void handleComment(CommentCreatedEvent commentEvent){
+        String comment = commentEvent.getComment();
+        String pullRequestTitle = commentEvent.getPullRequestTitle();
+        if ("Ignore this major".equalsIgnoreCase(comment)){
+            ignoreMajor(extractSettingFromComment(pullRequestTitle, "Major"));
+        }
+        else if ("Ignore this minor".equalsIgnoreCase(comment)){
+            ignoreMinor(extractSettingFromComment(pullRequestTitle, "Minor"));
```

Comment:
it could maybe be alitle bit clearer if you used the `extractSettingFromComment` method inside `ignoreMajor` and `ignoreMinor`